README.md

```diff
@@ -1,2 +1,29 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+![FlowChart](Flowchart.png)
+```
+SecretCode ← Generated Random Word;
+UserGuess ←  User's Selected Guess;
+
+If (SecretCode == UserGuess)
+{
+	Output << Correct;
+	Output << Terminal Unlocked;
+	Unlock Terminal;
+	End Program;
+}
+Else
+{
+	numberCorrect ← 0;
+	numberOfLetters ← 0;
+	For each letter in UserGuess { 				► i = letter number
```

Comment:
Would be better to have the definition of `i` in the code rather than in a comment. For example could write something like `For each letter index i` or more explicitly `For i = 0, 1, ..., UserGuess.length-1`